app/models/concerns/orchestration/dhcp.rb

```diff
@@ -135,7 +135,11 @@ def dhcp_record_name(record_mac)
   end
 
   def queue_dhcp
-    return unless (dhcp? || (old && old.dhcp?)) && orchestration_errors?
+    return unless dhcp? || (old && old.dhcp?)
+    unless orchestration_errors?
+      logger.warning("Not queueing DHCP: #{(errors + host.errors).full_messages.to_sentence}")
```

Comment:
It seems that ActiveModel errors cannot be added together (`undefined method `+' for #<ActiveModel::Errors:0x00000015624d40>.`), how about logging DHCP and Host errors in 2 different lines?